parser/src/main/java/com/microsoft/gctoolkit/parser/UnifiedG1GCParser.java

```diff
@@ -290,12 +290,12 @@ public void heapSize(GCLogTrace trace, String line) {
 
     public void heapRegionSize(GCLogTrace trace, String line) {
         regionSize = trace.getIntegerGroup(1);
-        G1GCForwardReference.setHeapRegionSize(regionSize);
     }
 
     //[15.316s][debug][gc,heap      ] GC(0)   region size 1024K, 24 young (24576K), 0 survivors (0K)
     //ignore this logging for now
     private void youngRegionAllotment(GCLogTrace trace, String line) {
+        regionSize = trace.getIntegerGroup(1) / 1024;
```

Comment:
I'd rather see forwardReference.setHeapRegionSize called here, even if the value doesn't change. I don't like that heapRegionSize gets treated differently as it makes code maintenance more difficult. Much easier to find where setHeapRegionSize is called, for example.